### DISCUSSION TITLE
Syncing config of dev with prod (table tracking and more)

### DISCUSSION THREAD

**Discussion by jonocodes** [hasura/graphql-engine#9490] [Question]
My goal is to have my open source project be runnable by a new user without having to interact with the Hasura console. I already have migrations and seeds in my repo and running via the CLI. But when setting the project up I need to enter the console to do several manual things. For example:
* choosing which tables are tracked
* choosing the name of the public/anon user
* setting table roles/permissions

Syncing metadata does not cover these cases. Is there a way to sync these, or set them via the cli?

I see these settings are accessible via the Metadata API, but perhaps there is a reason they are not in the cli?
https://hasura.io/docs/latest/api-reference/metadata-api/table-view/#metadata-pg-track-table

**Comment by meetzaveri** [comment]
Hey @jonocodes

Thanks for your feedback. I'll tag @scriptonist for his inputs, as he's one of the team members who has worked on CLI.

My take is that, tracking tables involve a good amount of payload to be sent in form of JSON which would contain contents related to tables and their customization. That's why API format is more preferable for track table functionality.
